include/circt/Dialect/RTL/Ops.h

```diff
@@ -8,12 +8,25 @@
 #define CIRCT_DIALECT_RTL_OPS_H
 
 #include "circt/Dialect/RTL/Dialect.h"
+#include "mlir/IR/FunctionSupport.h"
 #include "mlir/IR/OpImplementation.h"
+#include "mlir/IR/RegionKindInterface.h"
+#include "mlir/IR/SymbolTable.h"
 #include "mlir/Interfaces/SideEffectInterfaces.h"
 
 namespace circt {
 namespace rtl {
 
+// This holds the name, type, direction of a module's ports
+typedef std::tuple<StringAttr, Type, StringAttr> RTLModulePortInfo;
```

Comment:
This ends up being a little obfuse later, with all the std::get<>s.  Seems like some accessors would be nice, or even making this a class rather than a typedef?

Reply:
Agreed, please make this a `struct` to allow nicer access to it.